src/app/common/services/configuration-service.coffee

```diff
@@ -0,0 +1,14 @@
+angular.module("doubtfire.common.services.configuration", [])
+
+.constant('external_name', '/* @echo EXTERNAL_NAME */')
+#
+# Services for a configurable external name
+#
+.factory("configurationService", (external_name) ->
```

Comment:
Delete the factory to remove the service and update according to `sign-in.coffee` example -- only need to inject the constant above.